### DISCUSSION TITLE
[Bug] Clicking on the "System Tray" category in the properties window, the properties window crashes on Windows 10

### DISCUSSION THREAD

**Discussion by sdw4938** [valinet/ExplorerPatcher#1020] [General]
1. Open ExplorerPatcher's Properties window
2. Click the "System Tray" Category

ExplorerPatcher  22000.493.42.30
OS: Windows 10 Build 19043.1566
Event Viewer log:
Faulting application name: rundll32.exe_dxgi.dll, version: 10.0.19041.746, time stamp: 0xfb4a9a6b
Faulting module name: dxgi.dll, version: 22000.493.42.30, time stamp: 0x62245f45
Exception code: 0xc0000005
Fault offset: 0x0000000000044460
Faulting process id: 0x2cc4
Faulting application start time: 0x01d831a5ecb7f28c
Faulting application path: C:\Windows\system32\rundll32.exe
Faulting module path: C:\Windows\dxgi.dll
Report Id: c104396b-7aa2-4aa3-97f2-36231db4c218
Faulting package full name: 
Faulting package-relative application ID:

**Comment by valinet** [comment]
Tracked in https://github.com/valinet/ExplorerPatcher/issues/1013

Will be fixed in the next pre-release.

Thanks for the report.

**Reply by sdw4938** [reply]
You're Welcome
